src/error.rs

```diff
@@ -26,6 +26,8 @@ pub(crate) enum Error {
   FileOrderUnknown { text: String },
   #[snafu(display("I/O error at `{}`: {}", path.display(), source))]
   Filesystem { source: io::Error, path: PathBuf },
+  #[snafu(display("File parsing error: {}", source))]
+  FileParse { source: ignore::Error },
```

Comment:
This feels like a cop out, but with the ignore crate implementing it's own enum of different errors, I wasn't sure how to represent it properly without just deferring to that crate's Display implementation.